emails/models.py

```diff
@@ -126,6 +129,8 @@ def save(self, *args, **kwargs):
         if not self.server_storage:
             relay_addresses = RelayAddress.objects.filter(user=self.user)
             relay_addresses.update(description="", generated_for="", used_on="")
+        # TODO: any time a profile sets store_phone_log to False, delete the
```

Comment:
TODO: Write an issue to make sure we do this task. Bonus: reference the issue number in the comment.